DfE.FindInformationAcademiesTrusts/Extensions/OfstedRatingScoreExtensions.cs

```diff
@@ -26,8 +28,9 @@ public static string ToDisplayString(this OfstedRatingScore rating)
             OfstedRatingScore.RequiresImprovement => "Requires improvement",
             OfstedRatingScore.Inadequate => "Inadequate",
             OfstedRatingScore.None => "Not yet inspected",
+            OfstedRatingScore.NoJudgement => "No Judgement",
```

Comment:
```suggestion
            OfstedRatingScore.NoJudgement => "No judgement",
```
Should this be sentence cased?

Reply:
Moved comment to #641